README.md

```diff
@@ -131,3 +131,13 @@ The German government has asked SAP and Deutsche Telekom to develop the Corona-W
 [cwa-verification-portal]: https://github.com/corona-warn-app/cwa-verification-portal
 [cwa-verification-iam]: https://github.com/corona-warn-app/cwa-verification-iam
 [cwa-testresult-server]: https://github.com/corona-warn-app/cwa-testresult-server
+
```

Comment:
This was removed with https://github.com/corona-warn-app/cwa-app-ios/pull/1504/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L126 on purpose. @kaddaSz could you please take a look at this, when you are back from vacation?

Reply:
@Ein-Tim, @marcussc :  
Yes we removed it with the change to the [REUSE Tool](https://reuse.software/). For Android this switch wasn't done yet but is in preparation.

As it seems to be harder now to find the right informtation, I am fine with adding the Licensing information again to the Readme file, but with some small changes. Please check my inline comments below.

Reply:
I've pushed a commit which updates the file with your requested changes @kaddaSz